src/ReportGenerator.Core/Program.cs

```diff
@@ -17,7 +17,7 @@ public class Program
         /// <returns>Return code indicating success/failure.</returns>
         public static int Main(string[] args)
         {
-            args = args.Select(a => a.EndsWith("\"", StringComparison.OrdinalIgnoreCase) ? a.TrimEnd('\"') + "\\" : a).ToArray();
```

Comment:
not sure in which case we run into this scenario -> Didn't see happening in windows/*nix platform.